robyn/router.py

```diff
@@ -108,6 +116,9 @@ def add_route(  # type: ignore
         endpoint: str,
         handler: Callable,
         is_const: bool,
+        auth_required: bool,
+        openapi_name: str,
+        openapi_tags: List[str],
```

Comment:
why do middleware routes need to have open_api names?

Reply:
I don't think I've added openapi to middleware routes. MiddlewareRouter starts at line 266. These lines (120&121) are in Router.